### ISSUE TITLE
Documentation Transition: MkDocs + Markdown + PlatyPS

### ISSUE THREAD

**Issue by michaeltlombardi** [WahlNetwork/Vester#104]
Documentation should be easy to write, contribute to, and publish and allow for auto-generated reference documentation.

## Expected Behavior
Contributors to documentation should be able to write that documentation in markdown, the de facto documentation markup standard for PowerShell. They should also be able to update the documents without having to fiddle too much with configuration. The documentation should be deployable to ReadTheDocs or another static site via CI and include testing. The documentation should also support exporting the existing reference documentation for public functions so that that documentation is not being updated in multiple places.

## Current Behavior
Currently, the docs are manually written in ReStructured Text (RST) and [published](http://vester.readthedocs.io/en/latest/index.html) to ReadTheDocs. No reference documentation is included.

## Possible Solution
Use [mkdocs](http://www.mkdocs.org/) for documentation, allowing the contributors to write docs in markdown, theme and publish to readthedocs with minimal effort. Reference documentation can be generated from the module using [PlatyPS](https://github.com/PowerShell/platyPS). 

## Context
This would make contributing to documentation easier.

## Your Environment
* Module version used: Vester `1.0.1`
* Operating System: Windows `10.0.14393`
* PowerShell version: `5.1.14393.693`

**Comment by brianbunke** [comment]
We discussed this in Slack yesterday, and @michaeltlombardi may have time to assist with this transition.

I agree with the move from RST to Markdown. The one time in March I got motivated to do a full docs refresh, I got frustrated learning RST syntax, and the lack of an easy local preview option on my Win10 box.

I think our neglect in updating them speaks for itself: Any move toward ease of refresh (and a healthier doc update cadence) is welcome.
